deep-security-agent/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright:: 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+
+#######################
+# Mandatory Variables #
+#######################
+---
+# As you add agents to Deep Security, they are automatically made available for download through
```

Comment:
Please remove automatically since we still need to do something to make it available right?

Reply:
The wording "automatically" describes that agent packages can be download from manager without further configure once they were imported. I think we don't have to remove this wording.